packages/tools/pluggable-widgets-tools/configs/rollup-plugin-widget-typing.js

```diff
@@ -23,6 +23,10 @@ export function widgetTyping({ sourceDir }) {
             if (!firstRun) {
                 await runTransformation(sourceDir);
             }
+            await execShellCommand(
```

Comment:
Are you sure you cannot just use `execSync` here?

Reply:
Maybe, but I like promises as it's easier to understand and we have more control over what to do with `stdout` vs `stderr`

Reply:
I like promises as well as it's more clear we are waiting on something with `await`. FYI: `execSync` gives you the same control by returning `stdout` and throwing an Error with all details when it exits with a non-zero exit code.